install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -165,7 +165,16 @@ data:
         - mountPath: /etc/certs/
           name: istio-certs
           readOnly: true
+        {{ if .Values.global.sdsEnabled }}
+        - mountPath: /var/run/sds
+          name: sdsudspath
+        {{- end }}
       volumes:
+      {{ if .Values.global.sdsEnabled }}
+      - name: sdsudspath
+        hostPath:
+          path: /var/run/sds
+      {{- end }}
       - emptyDir:
           medium: Memory
```

Comment:
I agree eventually we need to add if/else after SDS works e2e works in master branch(tracked by https://github.com/istio/istio/issues/9035); secret file mount/sds have to co-exist for a while since making sds works e2e is incremental change(ex, sds works in ```gcp-identity``` branch today in a way that envoy's dynamic config(generate/distribute by pilot) in SDS format, whie bootstrap file still goes secret mount for now); I prefer change incrementally and keep this until having a basic flow works e2e in master first. 

To verify SDS works, pilot will only generated sdsconfig in LDS/CDS when meshconfig.sdsudspath is set (no secret files mount), that would let's know SDS actually works.